cssdb.json

```
@@ -504,6 +504,15 @@
       }
     ]
   },
+  {
+    "id": "overscroll-behavior",
+    "title": "`overscroll-behavior` Property",
+    "description": "CSS property to control the behavior when the scroll position of a scroll container reaches the edge of the scrollport",
```

Could we change this to drop the `CSS` explainer word and follow the `TYPE(S) for REASON(S)` style of other descriptions?

```suggestion
    "description": "Properties for controlling when the scroll position of a scroll container reaches the edge of a scrollport",
```